Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM golang:1.12
+FROM golang:1.12-alpine3.10 AS builder
```

Comment:
The building image also have to be changed to alpine as the binary is built with `CGO_ENABLED=1`.